components/form/builder/src/index.scss

```diff
@@ -52,8 +53,8 @@ $fz-form-builder-label: $fz-base !default;
     box-sizing: border-box;
     flex-grow: 1;
     margin-bottom: $m-l;
```

Comment:
Might set a variable for the margin bottom too. Its name can collision with the proposed

Reply:
@arnau-rius & @andresin87 
You're right. I'll upload a new proposal.
How about $m-form-builder-field-side?

Reply:
Maybe you can tokenize the whole margin property: `margin: $m-form-builder-field` so there is no need to change the variable and we have all 4 margins tokenized.